tools/wheels/LICENSE_win32.txt

```diff
@@ -879,3 +879,26 @@ may consider it more useful to permit linking proprietary applications with
 the library.  If this is what you want to do, use the GNU Lesser General
 Public License instead of this License.  But first, please read
 <http://www.gnu.org/philosophy/why-not-lgpl.html>.
+
+
+Name: libquadmath
+Files: scipy.libs\libopenblas*.dll
```

Comment:
Most of the PR looks similar to what Matti/Ralf did upstream, and so does this, but this looks incorrect because it is quadmath with OpenBLAS shared lib, and also if we statically linked I'm not sure it would be a dll vs. i.e., `.a`?

Reply:
I guess that is indeed what is in the distributed wheel, so maybe it is just the line about static linking below that confuses me re: DLL.